docs/markup-variables.md

```diff
@@ -86,12 +86,14 @@ asset Template extends AccordClause {
 
 the following instance text:
 ```md
-The penalty amount is 10.5% of the total value of the Equipment whose delivery has been delayed.
+The penalty amount is 10.5% of the total value of the Equipment whose delivery has been
+delayed.
 ```
 
 matches the template:
```

Comment:
it will not match if there is an extra line in the text but not in the template.